src/contexts/ContactsProvider.js

```diff
@@ -0,0 +1,375 @@
+import React, { useCallback, useContext, useState, useEffect } from 'react';
+import useLocalStorage from '../hooks/useLocalStorage';
+
+/**
+ * class User - 사용자 Data Model
+ * @namespace
+ * @typedef User
+ * @class 
+ * @type {object}
+ * @property {string} userId - 고유 사용자 ID. 사용자가 회원가입시 설정하며, 로그인시 사용. 프로그램 내에서 UNIQUE PRIMARY KEY로 사용
+ * @property {string} userName - 사용자 이름
+ * @property {string} statusMsg - 상태 메세지
+ * @property {object} lastActive - Javascript Date object. 마지막 활동 시각
+ */
+class User {
+  constructor(
+    userId,
+    userName,
+    statusMsg = '',
+    lastActive
+  ) {
+    this.userId = userId; //String
+    this.userName = userName; //String
+    if (lastActive) {
+      this.lastActive = lastActive; //Date
+    } else {
+      this.lastActive = new Date(); //Date
+    }
+    this.statusMsg = statusMsg; //String
+  }
+
+  /**
+   * 친구목록 등의 프로필 사진에서 현재 활동 중 표시부 색상(녹색/회색) 결정을 위해 사용
+   * @method getCurrentlyActive()
+   * @returns {boolean} currentlyActive - 사용자가 activeThreshold에서 설정한 시간 내에 활동이력이 있다면 true, 그렇지 않다면 false
+   */
+  getCurrentlyActive() {
```

Comment:
boolean을 반환하는 method는 일반적으로 is를 사용합니다.

```suggestion
  isCurrentlyActive() {
```

usage:
```jsx
YeongWoo.isCurrentlyActive()
```